examples/web-app/src/components/Utils.js

```diff
@@ -0,0 +1,305 @@
+import { UserFactory } from "nightfall-sdk";
+import {
+  erc1155ContractAddress,
+  erc20ContractAddress,
+  erc721ContractAddress,
+  erc20value,
+} from "./Constants";
+
+const clientApiUrl = process.env.CLIENT_API_URL;
+
+async function makeDeposit(e, nightfallMnemonic) {
+  e.preventDefault();
+  // Create a user with clientApiUrl and nightfallMnemonic
+  // Prevent default onClick behaviour that refreshes the page
+  try {
+    const nightfallUser = await UserFactory.create({
+      clientApiUrl,
+      nightfallMnemonic,
+    });
+
+    // Make a deposit for the user
+    const txReceipts = await nightfallUser.makeDeposit({
+      tokenContractAddress: erc20ContractAddress,
+      tokenErcStandard: "ERC20",
+      value: erc20value,
+    });
+    return txReceipts;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+async function makeTransfer(e, nightfallMnemonic) {
+  e.preventDefault();
+
+  // Create a user that will transfer funds
+  try {
+    const nightfallUserSender = await UserFactory.create({
+      clientApiUrl,
+      nightfallMnemonic,
+    });
+    // Create a user that will recieve funds
+    const nightfallUserRecepient = await UserFactory.create({
+      clientApiUrl,
+    });
+    const recepientAddress = nightfallUserRecepient.getNightfallAddress();
+
+    // Make a transfer to the nightfall address of the recipient
+    const txReceipts = await nightfallUserSender.makeTransfer({
+      tokenContractAddress: erc20ContractAddress,
+      tokenErcStandard: "ERC20",
+      value: erc20value,
+      recipientNightfallAddress: recepientAddress,
+      isOffChain: true,
```

Comment:
Remove all instances - will be set to `false` by default. Remember on-chain txs on Ganache are faulty, which is why your transfers aren't going through.